docs/algorithms/preference_comparisons.rst

```diff
@@ -50,10 +51,17 @@ Detailed example notebook: :doc:`../tutorials/5_train_preference_comparisons`
     reward_trainer = preference_comparisons.BasicRewardTrainer(
         preference_model=preference_model,
         loss=preference_comparisons.CrossEntropyRewardLoss(),
-        epochs=3,
+        epochs=10,
         rng=rng,
     )
 
+    # Several hyperparameters (reward_epochs, ppo_clip_range, ppo_ent_coef,
+    # ppo_gae_lambda, ppo_n_epochs, discount_factor, use_sde, sde_sample_freq,
+    # ppo_lr, exploration_frac, num_iterations, initial_comparison_frac,
+    # initial_epoch_multiplier, query_schedule) used in this example have been
+    # approximately fine-tuned to reach a reasonable initial experience. It's
+    # worth noting that we did not optimize all parameters those we did optimize
+    # may not be optimal.
```

Comment:
For brevity, I'd suggest skipping this comment in the .rst doc and consider putting it inside notebook instead.